### PR TITLE
Lower tf.IsFinite.

### LINKED ISSUE #47361: Support tf.IsFinite lowering
### 1. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Colab
- TensorFlow installation (pip package or built from source): pip
- TensorFlow library (version, if pip package or github SHA, if built from source): nightly

### 2. Code

```python
import tensorflow as tf

@tf.function
def func(x):
  return tf.is_finite(x)

converter = tf.lite.TFLiteConverter.from_concrete_functions([func.get_concrete_function(tf.TensorSpec(shape=(1, 2, 3)))])
converter.convert()
```

### 3. Failure after conversion

It will fail if flex is not enabled. However, `tf.is_inf()` is supported without flex because it is lowered to other ops in `lower_tf.cc`. It should be easy to support lowering pattern of `tf.is_finite`. Note that `tf.is_finite` is also being used in `tf.keras.layers.Softmax` with multi-axes reduction. I understand that I can enable flex to use it, but it should be a minimum work to lower it directly.

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/layers/advanced_activations.py#L328-L345

### PR DESCRIPTION
**Pull request by WindQAQ**
Fixes #47361. It allows to use `tf.is_finite` in TFLite without flex. `tf.is_finite` appears in `tf.keras.layers.Softmax` with multi-axes reduction.

/cc @abattery for visibility.

### CODE REVIEW

**Comment by abattery** [comment]
FYI, @thaink

**Comment by thaink** [comment]
This is almost the same as my PR. But It has not been able to get approved yet due to concern to affect other paths.

**Comment by WindQAQ** [comment]
Got it. Sorry that I forget you're working on this. Just pull one thing from my TODO list without a second look. So let me close it as there has been a pending one. Thanks!
